web/src/pages/apd/activities/oms/OutcomesAndMetrics/OutcomeAndMetricReview.js

```diff
@@ -19,16 +19,16 @@ const OutcomeAndMetricReview = ({
       {outcome || 'Outcome not specified'}
     </p>
     {metrics.length > 0 ? (
-      <p className="ds-u-margin-top--2">
-        <ul className="ds-c-list--bare subform__container">
+      <ul className="ds-c-list--bare subform__container">
```

Comment:
This isn't valid html (it wasn't originally either 🤣)

html list elements `<ul>` or `<ol>` should only contain `<li>` as its direct children.

so id recommend updating the jsx to reflect this structure-
```
<div class="subform__container ds-u-margin-top--2">
  <strong>Metrics:</strong>
  <ul class="ds-c-list--bare">
    <li class="ds-u-margin-bottom--2">[item 1]</li>
    <li class="ds-u-margin-bottom--2">[item 2]</li>
  </ul>
</div>
```

Reply:
😆 nice catch!